build/Linux+BSD/portable/copy-libs

```diff
@@ -93,6 +93,8 @@ getCrossPlatformDependencies() {
 
   copyQt plugins/sqldrivers/libqsqlite.so
 
+  copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so
```

Comment:
aha!  That makes sense!  (Seems a bit silly that Qt needs openGL 2.0 just for displaying a window :P !)